doc/example/00-azure-blob-storage-secret.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: v1
```

Comment:
Can you add the files to `/example` (as done in other repositories) instead of `/doc/example`?